.github/workflows/label-new-issues.yml

```diff
@@ -33,4 +33,4 @@ jobs:
             echo "Issue number must be a positive integer"
             exit 1
           fi
-          gh issue edit $ISSUE_NUMBER_INT --add-label 'pending-triage'
+          gh issue edit $ISSUE_NUMBER_INT --add-label 'pending-maintainer-response'
```

Comment:
I think transferred tickets are triggered by a different workflow, right?

Reply:
It looks to me like it's in this workflow. That's what the comments suggest, and it runs with
```
on:
  issues:
    types: [opened, reopened, transferred]
```